.cookiecutter/includes/pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+name = "h-testkit"
+dynamic = ["description", "version", "urls", "requires-python", "readme", "dependencies"]
```

Comment:
Once I add a `[project]` section to `pyproject.toml` them package build keeps failing unless I add all these here. These are all things that we set in `setup.cfg` instead of in `pyproject.toml`. I think we can probably move all these things into `pyproject.toml` and get rid of `setup.cfg` but that's for another day.